Source/Particles/Collision/BinaryCollision/NuclearFusion/ProtonBoronFusionInitializeMomentum.H

```diff
@@ -86,13 +86,13 @@ namespace {
         // Rest energy of alpha+beryllium
         constexpr amrex::ParticleReal E_rest_abe = (m_alpha + m_beryllium)*c_sq;
 
-        // These constexprs underflow in single precision because we use SI units and sometimes the
-        // code won't compile. Nuclear fusion module does not currently work with single precision
-        // anyways so these #ifdef are just here to make sure that the code always compiles with
-        // single precision.
+        // These constexprs underflow in single precision because we use SI units, which can cause
+        // compilation to fail or generate a warning. Nuclear fusion module does not currently work
+        // with single precision anyways so this #ifdef is here to make sure that compilation
```

Comment:
If this module does not work with SP, can you please add an always-assert with a message in runtime code in case a user calls it anyway? Or is there such an assert already?

Reply:
Yes, there is already such an assert.

https://github.com/ECP-WarpX/WarpX/blob/8eb3163d865fbabb22c227b8f1810db079cca8f3/Source/Particles/Collision/BinaryCollision/NuclearFusion/NuclearFusionFunc.H#L61-L63